os-eap-probes/2.0/added/livenessProbe.sh

```diff
@@ -16,26 +16,62 @@ COUNT=30
 SLEEP=5
 DEBUG_SCRIPT=false
 PROBE_IMPL="probe.eap.dmr.EapProbe probe.eap.dmr.HealthCheckProbe"
+INITIAL_SLEEP_SECONDS=5
+TMPLOC=${TMPDIR:-/tmp}
 
 if [ $# -gt 0 ] ; then
     COUNT=$1
 fi
 
-if [ $# -gt 1 ] ; then
-    SLEEP=$2
-fi
+# setting the first parameter to true/false, gets you the new default behaviour of
```

Comment:
Things are getting complicated. Usage section will be very helpful. 

I must say misusing COUNT parameter for boolean true of false (doesnt matter which) is very confusing

Why are we keeping old behaviour. For eap64 stream? Doesnt eap64 stream have own branch?

Reply:
@mchoma For the existing images (7.2.x on OCP 3.x) we felt that adding additional parameters for OCP 4.1 would be problematic for the user as they'd have to change depending on which version of OCP they were using. So we provided the true / false mechanism as a way to switch between these behaviours.

This is only for the 7.2.x images, the internal retry is dropped for CD 17 and later images.